example-web/app.json

```
@@ -0,0 +1,11 @@
+{
+  "name": "picker-web-example",
+  "displayName": "<%= project.name %> Example",
+  "expo": {
+    "name": "picker-web-example",
+    "slug": "picker-web-example",
+    "platforms": [
+      "web"
+    ]
```

```suggestion
    "name": "picker-web-example"
```

Expo CLI can do the rest <33